compute_endpoint/globus_compute_endpoint/endpoint/config/config.py

```diff
@@ -95,6 +95,9 @@ class Config(RepresentationMixin):
     display_name : str | None
         The display name for the endpoint.  If None, defaults to name
         Default: None
+
+    endpoint_setup : str | None
+        Command or commands to be run during the endpoint initialization process.
```

Comment:
We've discussed out-of-band the other half of this.  While we can't necessarily guarantee that an `endpoint_teardown` would execute, any of the reasons why it wouldn't fall under the "yeap, that's normal" category.  So, per the ticket and our discussion, I suggest adding that as well.